x-pack/test/security_solution_cypress/cypress/e2e/investigations/alerts/changing_alert_status.cy.ts

```diff
@@ -342,7 +338,7 @@ describe.skip('Changing alert status', { tags: ['@ess', '@skipInServerless'] },
         });
     });
 
-    it('Closes alerts', () => {
+    it.skip('Closes alerts', () => {
```

Comment:
t2 Analyst seems to have the ability to close alerts here. @MadameSheema is this behavior accurate?

Reply:
Solved it in an async way, seems that the t2 analyst has the privilege to close alerts.

Reply:
Are we going to maintain this test skipped?

Reply:
Nope, will remove it..and the one above it as that one is only passing currently since we are not waiting for the alerts to load after marking as Acknowledged